pkg/resources/saml2_integration.go

```diff
@@ -0,0 +1,543 @@
+		},
+	},
+	"saml2_post_logout_redirect_url": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "The endpoint to which Snowflake redirects users after clicking the Log Out button in the classic Snowflake web interface. Snowflake terminates the Snowflake session upon redirecting to the specified endpoint.",
+	},
+	"saml2_force_authn": {
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Description: "The Boolean indicating whether users, during the initial authentication flow, are forced to authenticate again to access Snowflake. When set to TRUE, Snowflake sets the ForceAuthn SAML parameter to TRUE in the outgoing request from Snowflake to the identity provider. TRUE: forces users to authenticate again to access Snowflake, even if a valid session with the identity provider exists. FALSE: does not force users to authenticate again to access Snowflake.",
+	},
+	"saml2_snowflake_issuer_url": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Computed:    true,
+		Description: "The string containing the EntityID / Issuer for the Snowflake service provider. If an incorrect value is specified, Snowflake returns an error message indicating the acceptable values to use.",
+	},
+	"saml2_snowflake_acs_url": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Computed:    true,
+		Description: "The string containing the Snowflake Assertion Consumer Service URL to which the IdP will send its SAML authentication response back to Snowflake. This property will be set in the SAML authentication request generated by Snowflake when initiating a SAML SSO operation with the IdP. If an incorrect value is specified, Snowflake returns an error message indicating the acceptable values to use.",
+	},
+	"saml2_snowflake_metadata": {
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "Metadata created by Snowflake to provide to SAML2 provider.",
+	},
+	"saml2_digest_methods_used": {
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "Digest methods used for this integration.",
+	},
+	"saml2_signature_methods_used": {
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "Signature methods used for this integration.",
+	},
+	"allowed_user_domains": {
+		Type: schema.TypeList,
+		Elem: &schema.Schema{
+			Type: schema.TypeString,
+		},
+		Optional:    true,
+		Description: "A list of email domains that can authenticate with a SAML2 security integration.",
+	},
+	"allowed_email_patterns": {
+		Type: schema.TypeList,
+		Elem: &schema.Schema{
+			Type: schema.TypeString,
+		},
+		Optional:    true,
+		Description: "A list of regular expressions that email addresses are matched against to authenticate with a SAML2 security integration.",
+	},
+	"comment": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "Specifies a comment for the integration.",
+	},
+	"created_on": {
```

Comment:
wrap the SHOW outpu into show_output map please

Reply:
Done